app/main.py

```diff
@@ -2,0 +1,46 @@
+
+class Animal:
+    def __init__(self, name: str,
+                 appetite: int,
+                 is_hungry : bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            self.is_hungry = False
+            print(f"Eating {self.appetite} food points...")
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self, name : str,
+                 is_hungry : bool = True) -> None:
+        super().__init__(name, appetite=3, is_hungry=is_hungry)
+
+    @staticmethod
+    def catch_mouse() -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self, name : str,
+                 is_hungry : bool = True) -> None:
+        super().__init__(name, appetite=7, is_hungry=True)
+
+    @staticmethod
+    def bring_slippers() -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(animals : list) -> None:
```

Comment:
The `feed_animals` function is defined to return `None`, but it actually returns `total_food`. The return type in the function signature should be updated to reflect that it returns an `int`.